plugins/sidebar/overlays/progress.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+
+const React = require('react');
+
+class Progress extends React.Component {
+
+  render(){
+
+    const setComplete = this.props.setComplete + '%';
+
+    const barStyle = {
```

Comment:
these shouldn't be in a render, react-style lets you join static with dynamic styles